scripts/pbsserverhooks.sh

```diff
@@ -1,27 +1,13 @@
 #!/bin/bash
 # Add hooks
 
-log_analytics_account=$1
-log_analytics_key=$2
-
 cd /tmp
-sudo yum install -y git
-git clone https://github.com/JonShelley/pbs_hooks.git
-cd pbs_hooks/azure
-
-
+wget https://raw.githubusercontent.com/Azure/azurehpc/master/pbshooks/setup_jobdir.py
```

Comment:
it would be much better if we didn't use wget here.  if you were to change the branch (or if someone cloned the repo) and made changes, this would not give the right results

Reply:
@edwardsp I have removed the wget now